tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
```

Comment:
@smarnach Thanks - and yes, it would be good to copy it from there. To avoid getting out of sync, maybe it could be done by wget'ing it from the master of the opencraft repo, this way it would not require duplicating the file in this repo?